src/GraphQL/GraphQL.csproj

```diff
@@ -27,7 +27,7 @@
   <ItemGroup>
     <PackageReference Include="GraphQL-Parser" Version="4.1.1" />
     <PackageReference Include="Microsoft.CSharp" Version="4.5.0" />
-    <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="2.0.0" />
+    <PackageReference Include="Microsoft.Extensions.DependencyInjection.Abstractions" Version="2.0.0" />
```

Comment:
In fact, move to `.Abstractions` does not change much in comparison with general remarks in this PR.